solutions/usaco-1060.mdx

```diff
@@ -61,8 +61,8 @@ import java.util.*;
 import java.io.*;
 	
 class Main {
-	public static void main(String[] args) {
-		Kattio io = new Kattio()
+	public static void main(String[] args) throws IOException{
```

Comment:
why throw the exception? im p sure console io doesnt need it.

Reply:
think it does?

Reply:
it does need it it wont work without it

Reply:
i just checked, u do not need it

Reply:
^^ I can confirm, Kattio without file i/o doesn't need a `throws` statement

Reply:
removed throws IOE exception but i still need to add the ";"